Tools/test/TrackingTreer/treer.py

```diff
@@ -18,27 +20,34 @@
 process.load('JMTucker.Tools.UnpackedCandidateTracks_cfi')
 process.load('JMTucker.Tools.UpdatedJets_cff')
 process.load('JMTucker.Tools.WeightProducer_cfi')
+#process.load('JMTucker.Tools.LeptonCuts_cfi')
 
 process.selectedPatJets.src = 'updatedJetsMiniAOD'
 process.selectedPatJets.cut = process.jtupleParams.jetCut
 
 process.tt = cms.EDAnalyzer('TrackingTreer',
                             process.jmtNtupleFillerMiniAOD,
-                            track_cut_level = cms.int32(1), # -1 = all, 0 = pt & pix & strip, 1 = 0 + min_r, 2 = 1 + nsigmadxybs
+                            track_cut_level = cms.int32(0), # -1 = all, 0 = pt & pix & strip, 1 = 0 + min_r, 2 = 1 + nsigmadxybs,
```

Comment:
@abbywarden Maybe I don't know the use case, but did we intentionally change this one?
(I'll make the same comment as before that there might be some cleanup of unused commented out stuff that we could do here)